app/controllers/admin/block_controller.rb

```diff
@@ -2,23 +2,23 @@ class Admin::BlockController < AdminController
   before_action :load_user, :admin_user
 
   def update
+    profile = {user_id: @user.id}
+    BlockWorker.perform_async(profile)
```

Comment:
```suggestion
    BlockWorker.perform_async @user.id
```
cần gì gán rồi trong worker lại phải lấy ra e

Reply:
vang anh